source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ImageSetRenderer.java

```diff
@@ -94,11 +94,11 @@ public View render(
 
         if (imageSet.GetHeight() == HeightType.Stretch)
         {
-            viewGroup.addView(horizFlowLayout, new LinearLayout.LayoutParams(LinearLayout.LayoutParams.MATCH_PARENT, LinearLayout.LayoutParams.MATCH_PARENT, 1));
+            viewGroup.addView(horizFlowLayout, new LinearLayout.LayoutParams(LinearLayout.LayoutParams.WRAP_CONTENT, LinearLayout.LayoutParams.MATCH_PARENT, 1));
```

Comment:
what's he justification for this?

Reply:
Good call out - this was leftover from my initial investigation and doesn't have an impact. Just removed it